frontend/webapp/core/actions.py

```diff
@@ -12,122 +12,36 @@
 log = logging.getLogger("webapp_logger")
 
 
-class Resolve_hijack:
-    def __init__(self, hijack_key, prefix, type_, hijack_as):
-        self.connection = None
-        self.hijack_key = hijack_key
-        self.prefix = prefix
-        self.type_ = type_
-        self.hijack_as = hijack_as
-        self.init_conn()
-        self.hijack_exchange = Exchange(
-            "hijack-update", type="direct", durable=False, delivery_mode=1
-        )
-
-    def init_conn(self):
-        try:
-            self.connection = Connection(RABBITMQ_URI)
-        except BaseException:
-            log.exception("Resolve_hijack failed to connect to rabbitmq.")
-
-    def resolve(self):
-        log.debug("send resolve hijack message with key: {}".format(self.hijack_key))
-        with Producer(self.connection) as producer:
-            producer.publish(
-                {
-                    "key": self.hijack_key,
-                    "prefix": self.prefix,
-                    "type": self.type_,
-                    "hijack_as": self.hijack_as,
-                },
-                exchange=self.hijack_exchange,
-                routing_key="resolved",
-                priority=2,
-            )
-
-
-class Mitigate_hijack:
-    def __init__(self, hijack_key, prefix):
-        self.connection = None
-        self.hijack_key = hijack_key
-        self.prefix = prefix
-        self.mitigation_exchange = Exchange(
-            "mitigation", type="direct", durable=False, delivery_mode=1
-        )
-        self.init_conn()
-
-    def init_conn(self):
-        try:
-            self.connection = Connection(RABBITMQ_URI)
-        except BaseException:
-            log.exception("Resolve_hijack failed to connect to rabbitmq.")
-
-    def on_response(self, message):
-        if message.properties["correlation_id"] == self.correlation_id:
-            self.response = message.payload
-
-    def mitigate(self):
-        log.debug("sending mitigate message")
-        with Producer(self.connection) as producer:
-            producer.publish(
-                {"key": self.hijack_key, "prefix": self.prefix},
-                exchange=self.mitigation_exchange,
-                routing_key="mitigate",
-                priority=2,
-            )
-
-
-class Ignore_hijack:
-    def __init__(self, hijack_key, prefix, type_, hijack_as):
-        self.connection = None
-        self.hijack_key = hijack_key
-        self.prefix = prefix
-        self.type_ = type_
-        self.hijack_as = hijack_as
-        self.init_conn()
-        self.hijack_exchange = Exchange(
-            "hijack-update", type="direct", durable=False, delivery_mode=1
+def rmq_hijack_action(obj):
+    log.info(
+        "Send '{0}' hijack message with key: {1}".format(
+            obj["action"], obj["payload"]["key"]
         )
+    )
 
```

Comment:
you should also add an assertion here with what the obj should include (e.g., payload, routing_key, exchange, etc.)